File: Making API CALLS/call.js
const response = await fetch("https://api.openai.com/v1/chat/completions",
                              { Method : "POST",
                                headers: { "Content-Type" : "application/json",
                                    "Authorization" : 'Bearer ${OPENAI_API_KEY}'
                                        },
                                        body : JSON.stringify({
                                            model : "gpt-4o",
                                            messages:[{role:"user",content:"Who is neyy"}]
                                        })
                              });
                              const data =await response.json();
                              console.log(data);

File: Making API CALLS/call.py
import requests

response = requests.post("https://api.openai.com/v1/chat/completions",
                         headers={"Authorization": f"Bearer YOUR_KEY"},
                         json={
                             "model": "gpt-4o",
                             "messages":[{"role":"user","content":"Who is Ronlado"}]
                         })
print(response.json())